client/src/components/Modals/SignUpModal.tsx

```diff
@@ -0,0 +1,33 @@
+import Button from '@mui/material/Button';
+import React from 'react';
+
+interface SignUpModalProps {
+  closeModal: () => void;
+  classes: {
+    outerShell: string;
+    paper: string;
+    content: string;
+    header: string;
+    loginButton: string;
+    buttonContainer: string;
+    closeButton: string;
+  };
+}
+
+const SignUpModal = React.forwardRef<HTMLDivElement, SignUpModalProps>(
+  ({ closeModal, classes }, ref) => {
+    const handleCloseModal = () => {
+      closeModal();
+    };
+
+    return (
+      <div>
+        <h2>Sign Up</h2>
+        {/* ... SIGNUP CODE ... */}
```

Comment:
@EtoKruto , the sign up modal seems to be incomplete.

Reply:
Left it incomplete for Dennis' story as we were supposed to finish together but he is currently moving so didn't have time to finish the Sign-Up. I'll comment the return code inside the SignUp JSX out